### ISSUE TITLE
Normalize CSS colors for IE6-8

### ISSUE THREAD

**Issue by JamesMGreene** [JamesMGreene/qunit-assert-html#2]
**TL;DR:** This addon currently does not normalize certain CSS style properties (e.g. color values) in IE < 9.

IE9+ and other browsers normalize the value the user sets. IE8- leaves the value as exactly what the user specified, even if error correction behavior handles it differently. For a little more info, see jquery/qunit#368.

**Implementation notes for fixing:**
- http://www.sophox.com/wordpress/?p=833
- http://www.sophox.com/wordpress/?p=882
- http://stackoverflow.com/questions/11839088/get-real-value-of-a-css-color-property-in-script
- http://blog.stchur.com/2006/06/21/css-computed-style/
- http://msdn.microsoft.com/en-us/library/ie/ms535231(v=vs.85).aspx
